### DISCUSSION TITLE
How to use cilk_reducer --> write data to file

### DISCUSSION THREAD

**Discussion by mohandotyes** [OpenCilk/www.opencilk.org#198] [Q&A] [answered]
Hi Team,
How to use cilk_reducer -->> write data to file. 
How to implement void identity(void *view) and void reduce(void *left, void *right);

GOAL: Write data to file without race condition(using cilk_reducer)

**Comment by neboat** [comment, marked as answer]
I believe I answered your question on StackOverflow: https://stackoverflow.com/questions/75155198/how-to-use-cilk-reducer-variable-to-write-data-into-a-file-without-race-conditio/75203198#75203198

**Reply by mohandotyes** [reply]
Thank you @neboat it worked.
